src/modules/apps.js

```diff
@@ -281,4 +282,57 @@ export default {
       )
     },
   },
+  settings: {
+    description: 'Get app settings',
+    requiredArgs: 'app',
+    optionalArgs: 'field',
+    handler: async (app, field) => {
+      try {
+        let response = await appsClient().getAppSettings(
+            getAccount(), getWorkspace(), app)
+        if (typeof field === 'object') {
+          console.log(response)
+        } else {
+          let jsonPath = '$.' + field
+          console.log(jp.value(response, jsonPath))
+        }
+      } catch (err) {
+        log.error(`${err.name}: ${err.message || err.status}`)
+      }
+    },
+
+    set: {
+      description: 'Set a value',
+      requiredArgs: ['app', 'field', 'value'],
+      handler: async (app, field, value) => {
+        try {
+          let jsonPath = '$.' + field
+          let patch = {}
+          jp.value(patch, jsonPath, value)
```

Comment:
@rdumont `path` cant receive the value from `jp.value`? This kind of mutation is quite awful...